calcom/cal.com
# fix: Members count when `team` slug is same as `org` slug

## What does this PR do?
Fixes #12123


## Type of change

## How should this be tested?
- Create a team within an org
- Add a few members in it
- Rename the team slug to be same as org slug.
- Go to org profile page(calcom.cal.local:3000) and see that the team `calcom` shows 0 members. This bug is fixed here.

## Mandatory Tasks


## Checklist
- I haven't added tests that prove my fix is effective or that my feature works
